### PR TITLE
update playbooks

### PR DESCRIPTION
**Pull request by VladimirWork**
Signed-off-by: Vladimir Shishkin <vladimir.shishkin@dsr-corporation.com>

### CODE REVIEW

File: scripts/ansible/pool_get_logs.yml [hunk 1]
```diff
@@ -30,3 +33,27 @@
       src: /var/log/indy/sandbox/
       dest: ./logs
```
Comment (anikitinDSR) [inline scripts/ansible/pool_get_logs.yml]: if it possible, please change to absolute path instead of relative

Comment (VladimirWork) [reply scripts/ansible/pool_get_logs.yml]: Each scripts user forks indy-node in different place so probably it's better to keep relative paths for local machine.

**Comment by VladimirWork** [comment]
test this please

**Comment by VladimirWork** [comment]
test this please

**Pull request merged**